kartograf/coverage.py

```diff
@@ -31,7 +32,11 @@ def coverage(map_file, ip_list_file):
             ip = ipaddress.ip_address(line.rstrip('\n'))
             addrs.append(int(ip))
         except ValueError:
-            continue
+            print(f"""
+                  Invalid IPv4/IPv6 address provided: {line}.
+                  Please remove and re-run.
+                  """)
+            sys.exit()
```

Comment:
Hm, I think I would have just raised an error with the message instead of doing it like this. Did you go this route to make a nicer output for the user?

Reply:
yea fair point, no particular reason I went this route. Updated.